packages/web-react/src/hooks/useCancelEvent.ts

```diff
@@ -1,6 +1,12 @@
-import { useCallback, useEffect, MutableRefObject } from 'react';
-
-const EVENT_CANCEL = 'cancel';
+import { MutableRefObject, useCallback, useEffect } from 'react';
+
+/**
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/cancel_event
+ * @see https://jira.lmc.cz/browse/DS-1108
+ * Here should be used `cancel` event instead of `close` event.
+ * But when dialog is closed using `escape` key, `cancel` event is not fired.
+ */
+const EVENT_CANCEL = 'close';
```

Comment:
I was looking at this option to change `cancel` for close`, but now if you close modal with close button on with backdrop click `onClose` method is called twice - from button/backdrop and form close event. Are we ok with that?